src/app/clinic/clinic_facade.hpp

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include "../calendar/calendar.hpp"
```

Comment:
Adjust CMake and remove `"../x"`, same as #99

Reply:
Added